### PR TITLE
Add a default app.yaml

### PR DESCRIPTION
**Pull request by daveconde**
Fixes: #83

### CODE REVIEW

File: charts/jx-app-jacoco/templates/app.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: jenkins.io/v1
+kind: App
+metadata:
+    name: {{ template "fullname" . }}
+    labels:
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec: {}
```
Comment (hferentschik) [inline charts/jx-app-jacoco/templates/app.yaml]: Let's add a newline to keep GitHub happy ;-)

**Review by hferentschik — CHANGES_REQUESTED** [review]
I did not know about this file. Cool to add it. 

What's about sticking to https://www.conventionalcommits.org/en/v1.0.0-beta.3/ for writing commit messages? 

Also I am in favor of creating issues for each change. Not sure what you think about this?

**Comment by daveconde** [comment]
> I did not know about this file. Cool to add it.
> 
> What's about sticking to https://www.conventionalcommits.org/en/v1.0.0-beta.3/ for writing commit messages?

I haven't seen that before. I'll try and stick with that :)

> 
> Also I am in favor of creating issues for each change. Not sure what you think about this?

Sure I'll open an issue for both PRs so they are tracked. 

Let's hold this one until Pete is back, I'd like to discuss this approach vs automatically adding an App resource if one doesn't exist.

**Comment by hferentschik** [comment]
> I haven't seen that before. I'll try and stick with that :)

Thanks. It's a bit new to me as well. At least in this format. I am a strong believer in properly formatted commit messages, but we used a different format in the projects we worked on.

@pmuir mentioned conventional commits and that's fine with me. Especially since it also aids automated changelog generation.

> Sure I'll open an issue for both PRs so they are tracked.

+1 :-)

> Let's hold this one until Pete is back, I'd like to discuss this approach vs automatically adding an App resource if one doesn't exist.

Absolutely.

**Comment by pmuir** [comment]
> this approach vs automatically adding an App resource if one doesn't exist.

Yes, we need to add an App resource automatically if one doesn't exist, and we also need to handle the case where the App adds more than one App resource (what does that mean?).

**Review by hferentschik — CHANGES_REQUESTED** [review]
Can we not squash that into a single commit with a proper commit message?

**Comment by daveconde** [comment]
> Can we not squash that into a single commit with a proper commit message?

Looking better?

**Review by hferentschik — APPROVED** [review]

**Pull request merged**